en/02_Developer_Guides/00_Model/11_Scaffolding.md

```diff
@@ -113,6 +105,9 @@ class MyDataObject extends DataObject
 }
 ```
 
+> [!WARNING]
+> If you define a `searchable_fields` configuration, *do not* specify fields that are not stored in the database (such as methods), as this will cause an error.
```

Comment:
Moved this from above to satisfy [MD028](https://github.com/DavidAnson/markdownlint/blob/main/doc/md028.md) - can't have two block quotes immediately after one another.